aria-practices.html

```diff
@@ -53,23 +53,39 @@
+      The ARIA Authoring Practices Guide (APG) explains how to make rich internet applications accessible to users of assistive technologies and keyboard interfaces by developing web sites that properly convey accessibility semantics and implement common keyboard conventions.
+      It also provides guidance on some closely related topics, such as supporting operating system settings for high contrast and reduced motion.
     </p>
     <p>
-      Languages used to create rich and dynamic web sites, e.g., HTML, JavaScript, CSS, and SVG, do not natively include all the features required to make sites usable by people who use assistive technologies (AT) or who rely on keyboard navigation.
+      Accessibility semantics are the meaning of user interface elements that need to be conveyed to assistive technology users in order for those users to understand and use the elements.
+      For example, people who can visually perceive a search icon button understand the element can be activated to perform a search by the way it is styled and positioned.
+      To make that icon accessible to a screen reader user, one of the semantics that needs to be communicated is that the element represents a button, and there are several ways that semantic can be conveyed to assistive technologies in code.
+      In addition, for the button to be accessible via a keyboard, it needs to be keyboard focusable, and it is commonly expected that when focused, pressing <kbd>Enter</kbd> or <kbd>Space</kbd> will activate it.
+      The APG describes how to convey accessibility semantics and implement keyboard accessibility for many common design patterns, ranging from typical buttons and popup menus to complex tree grids.
+      It also explains essential practices related to these patterns, such as how to convey web page structure with ARIA landmark regions and effectively utilize the many ways of encoding accessible names.
+    </p>
+    <p>
+      The APG is organized into two major sections: patterns and practices.
+      Each pattern explains how to make a common user interface element, such as a button, menu, or dialog, accessible, and provides functional example implementations of the pattern.
+      The practices section gives in-depth explanation of how to satisfy a variety of accessibility needs that surface when making rich internet application experiences accessible.
+      For instance, the practices section on providing accessible names and descriptions gives detailed description of seven different naming techniques as well as a table providing guidance for naming more than 80 types of elements.
+    </p>
+    <p>
+      The APG assumes basic understanding of web development, especially of HTML and CSS.
+      It may be particularly useful to designers and engineers who are working on site design systems and component libraries.
+      However, it is important to understand that the APG covers only a portion of what is needed to create accessible experiences.
```

Comment:
Swap "APG covers only a portion" to "APG only covers a portion" (reads with a little more emphasis, imo)